.rubocop.yml

```diff
@@ -89,8 +89,8 @@ Layout/DotPosition:
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains
   EnforcedStyle: trailing
   SupportedStyles:
-  - leading
-  - trailing
+    - leading
```

Comment:
should we lint these yaml files? 😬 🤔

Reply:
> should we lint these yaml files? 😬 🤔

Seeing as we already do for other YAML files, I'd say we may as well!

https://github.com/18F/identity-idp/blob/bf8e49e5e706562f972f3e9ac20fb0659ea08f99/Makefile#L61-L62

Reply:
yes but we probably want to normalize and maybe skip the alphabetization?

Reply:
How does 05590d7ea78667700d3789dc6fb1bd0a345728ed and 18f1fe8647f0e4c10a0d8fb5aaed2a9c6357e47e look to you?

Reply:
Yes! But I'd make the option `skip-sorting-keys` or something? because we're still doing *some* formatting as part of the normalization

Reply:
Hm, yeah, I guess "formatting" is a bit ambiguous in the package, since it separately runs a set of visitors to transform content, in addition to the Prettier pretty-printing. There's also the [ellipsis / smartquotes normalization](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/normalize-yaml/visitors/format-content.js) which we probably don't need for this, which is why I avoided just the simple disable flag. Maybe it's fine to just have two flags for disabling (a) sorting and (b) content replacements. I'll try that.